app/views/devise/shared/_links.html.erb

```diff
@@ -1,3 +1,14 @@
+<%- if devise_mapping.omniauthable? %>
+  <%- resource_class.omniauth_providers.each do |provider| %>
+    <%= button_to(omniauth_authorize_path(resource_name, provider),
+                  class: 'btn btn-primary', id: "#{provider}-signin" ,
+                  method: :post,
+                  data: {disable_with: "<i class='fa fa-spinner fa-spin'></i> Logging in...", turbo: "false"}) do %>
```

Comment:
If I understand this correctly, the "logging in" text will be shown on the "Sign in with OIDC" button after clicking it, but I've been never able to see it, even with throttled requests to the OIDC server.

But if it should be visible in some cases, it would be nice if it was localized.

Reply:
I simply removed this text now, I think the icon suffices. I honestly also don't know when this should be shown.